app/controllers/api/v1/mixins/ace_mixin.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1
+    module Mixins
+      module ACEMixin
+        def ace_ids(verb, klass)
+          permission = "#{ENV['APP_NAME']}:#{klass.table_name}:#{verb}"
```

Comment:
verb alone should be sufficient, unless you want to match the permission from RBAC service